src/lv_message.h

```diff
@@ -16,6 +18,7 @@ namespace grpc_labview
     {
     public:
         LVMessage(std::shared_ptr<MessageMetadata> metadata);
+        LVMessage(std::shared_ptr<MessageMetadata> metadata, bool use_hardcoded_parse, bool skipCopyOnFirstParse);
```

Comment:
It may be easier to make this a global config.

Reply:
fixed using feature toggle